### DISCUSSION TITLE
Obtain nordigen literal responses given to FIDI

### DISCUSSION THREAD

**Discussion by CosmoRing** [firefly-iii/firefly-iii#6015] [Firefly III] [answered]
I'm having a problem with Nordigen returning the same transaction multiple times with different external_id and as mentioned on #5906 

> If your bank says it's two transactions (two different ID's) Firefly III will treat them as such. 

Is there any configuration i can change or volume i can mount to get the literal response that nordigen returns to the FIDI? My objective is to obtain that information and to do some custom logic (independent from Firefly III and FIDI) to determine what i consider to be duplicated and use Firefly III API to manipulate it (delete or add a custom tag or anything i want).

**Comment by JC5** [comment, marked as answer]
The log files report the original transactions. Check out your docker container logs or storage/logs
